sl_server.h

```diff
@@ -0,0 +1,82 @@
+#include "Backend.grpc.pb.h"
+#include "Backend.pb.h"
+#include "KeyValueStore.grpc.pb.h"
+#include "KeyValueStore.pb.h"
+#include "ServiceLayer.grpc.pb.h"
+#include "ServiceLayer.pb.h"
+#include "kvs_client.h"
```

Comment:
I think your project's .h files go at the end of the includes

Reply:
fixed.